### PR TITLE
feat: update theme

### LINKED ISSUE #14: Refactor colors based on provided color system in Figma
Finally some names and values to base on

image

### PR DESCRIPTION
**Pull request by jeydpeaschocolatebar**
Closes #14

### CODE REVIEW

File: src/assets/styles/themes.css [hunk 1]
```diff
@@ -11,23 +12,22 @@
...
     --password-text-color: 14, 163, 246;
     --placeholder-text-color: 204, 204, 204;
     --entry-cost-text-color: 204, 204, 204;
-    --table-header-text-on-background-color: 221, 221, 221;
+    --table-header-text-on-background-color: 205, 196, 233;
     --table-header-background-color: 57, 48, 83;
 
-    color: rgb(var(--text-on-background-color));
-    background-color: rgb(68, 60, 104);
+    @apply text-on-background bg-secondary-background;
```
Comment (cerinoligutom) [inline src/assets/styles/themes.css]:
```suggestion
```

This line is essentially the same as its ancestor so this isn't needed.

File: src/assets/styles/themes.css [hunk 2]
```diff
@@ -1,5 +1,6 @@
 :root {
   --background-color: 255, 255, 255;
+  --secondary-background-color: 233, 235, 238;
```
Comment (cerinoligutom) [inline src/assets/styles/themes.css]:
```suggestion
  --background-secondary-color: 233, 235, 238;
```

Modifiers/Variants after. Rename the variable for the dark classes and tailwind config as well.

**Review by cerinoligutom — CHANGES_REQUESTED** [review]

File: src/assets/styles/themes.css (hunk 1 above)
Comment (jeydpeaschocolatebar) [reply src/assets/styles/themes.css]: Line removed

File: src/assets/styles/themes.css (hunk 2 above)
Comment (jeydpeaschocolatebar) [reply src/assets/styles/themes.css]: Variable renamed

**Comment by jeydpeaschocolatebar** [comment]
@zeferinix pushed the requested changes, thank you.

**Review by cerinoligutom — APPROVED** [review]

**Pull request merged**